src/usdm-client.ts

```diff
@@ -113,131 +113,4 @@ export class USDMClient extends BaseRestClient {
       (response) => response.serverTime,
     );
   }
-
-  /**
-   *
-   * Portfolio Margin Pro Endpoints
-   *
-   **/
-
-  getPortfolioMarginProAccountInfo(params: {
-    asset: string;
-  }): Promise<PortfolioMarginProAccountInfo> {
-    return this.getPrivate('fapi/v1/pmAccountInfo', params);
-  }
-
-  /**
-   *
-   * Broker Futures Endpoints
-   *
-   **/
-
-  // 1 == USDT-Margined, 2 == Coin-margined
```

Comment:
Moved these to v2, but I think they might soon be obsolete - as I saw, all broker endpoints are using /broker/ url, so I guess this wont work anymore. I didnt test, we can check later and mark deprecated. ANyways, all of it is in v2, at the bottom :)